get-started/mcp/sdk-integrations/openai-agents.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Integrate OpenAI Agents SDK with kluster Verify
+description: This guide walks you through integrating OpenAI Agents SDK with kluster Verify to build reliable AI agents with real-time hallucination detection and fact-checking capabilities.
+---
+
+# Integrate OpenAI Agents SDK with kluster Verify
+
+[OpenAI Agents SDK](https://openai.github.io/openai-agents-python/){target=_blank} provides a powerful framework for building AI agents that can use tools, maintain context, and interact with external services. By integrating with [kluster Verify](/get-started/verify/overview/){target=_blank}, you can create agents that not only generate responses using [kluster.ai's](https://www.kluster.ai/){target=_blank} language models but also perform real-time reliability checks to detect hallucinations and validate factual claims with internet-sourced verification.
+
+This guide demonstrates how to integrate the `Agent` and `Runner` classes from the OpenAI Agents SDK with kluster.ai's API and MCP server, then walks through building an interactive chatbot with kluster Verify's hallucination detection capabilities.
+
+## Prerequisites
+
+--8<-- 'text/kluster-api-onboarding.md'
+- **Python 3.9+** with pip installed.
+- **[A Python virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/){target=_blank}** - this is optional but recommended. Ensure that you enter the Python virtual environment before following along with this tutorial.
+- **OpenAI Agents SDK packages installed** - install the [`openai-agents` packages](https://github.com/openai/openai-agents-python){target=_blank}:
+
+    ```bash
+    pip install "openai>=1.93.0" "openai-agents>=0.1.0" "mcp[cli]>=1.10.1"
+    ```
+
+- **MCP enabled**: Via [platform](https://platform.kluster.ai){target=_blank} or [API](/get-started/mcp/cloud/api/).
```

Comment:
```suggestion
- **MCP enabled**: This can be done via the [platform](https://platform.kluster.ai){target=_blank} or the [API](/get-started/mcp/cloud/api/){target=\_blank}.
```